Gemfile

```diff
@@ -10,7 +10,8 @@ gem 'puma', '~> 5.0'
 gem 'rack-cors', require: 'rack/cors'
 gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
 gem 'slack-notify'
-gem 'devise_token_auth'
+# gem 'devise_token_auth'
```

Comment:
do we need to delete this?

Reply:
No, I'd like to keep this comment around for now.